sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java

```diff
@@ -1703,7 +1780,7 @@ private static BasicDocumentRevision getFullRevisionFromCurrentCursor(Cursor cur
             parent = cursor.getLong(7);
         } else if(cursor.columnType(7) == Cursor.FIELD_TYPE_NULL) {
         } else {
-            throw new IllegalArgumentException("Unexpected type: " + cursor.columnType(7));
+            throw new IllegalArgumentException("Unexpected type: " + cursor.columnType(7)); //TODO Should be straight Runtime Exception? Definitely not an argument
```

Comment:
I don't know why, as far as I can tell from the schema we only add columns, so I assume it was to guard against programming errors